### PR TITLE
button css updated

### PR DESCRIPTION
**Pull request by rupeshgupta369**
Please give your reviews

### CODE REVIEW

**Comment by rupeshgupta369** [comment]
lorem

File: homepage.css [hunk 1]
```diff
@@ -75,10 +75,10 @@ body{
 }
 
 #homepage-buttons button {
-    font-size: 1.8em;
```
Comment (jay-rajput30) [inline homepage.css:78]: refactor: using consistent spacing . it would be better to use rem, em instead of px values.

**Review by manan228 — COMMENTED** [review]
Looks good

**Comment by rupeshgupta369** [comment]
refacor: using consistent spacing

**Pull request merged**